src/mobile-token/utils.ts

```diff
@@ -71,10 +72,7 @@ export const getSdkErrorTokenIds = (err: any): string[] =>
  * from https://github.com/entur/abt-mobile-client-sdk/pull/368
  */
 export const parseBffCallErrors = (error: any) => {
-  if (
-    error instanceof TypeError &&
-    error.message === 'Network request failed'
-  ) {
+  if (axios.isAxiosError(error) && !error.response) {
```

Comment:
@gorandalum is this okay?

Reply:
Take a look at `getAxiosErrorType` to see if that can be of help to make it a little clearer.

Reply:
updated :D